templates/footer.hbs

```diff
@@ -38,12 +38,12 @@
         <h2 class="govuk-visually-hidden">Support links</h2>
         <ul class="govuk-footer__inline-list">
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" href="https://help.apprenticeships.education.gov.uk/hc/en-gb/articles/360008941440-Cookies">
+            <a class="govuk-footer__link" href="/hc/en-gb/articles/{{settings.article_id_cookies}}-Cookies">
```

Comment:
Thanks for changing this - it also needs the corresponding change to the manifest.